### ISSUE TITLE
Benchmarks needed

### ISSUE THREAD

**Issue by karwa** [apple/swift-algorithms#110]
As I mentioned in #109, a benchmark suite is important for ensuring a stable level of performance.

Personally, I've been using https://github.com/google/swift-benchmark - it's a little rough around the edges, but it produces decent reports on all platforms and the dynamic iterations features works really well for getting stable readings out of small code snippets. I'd encourage Apple (or Swift.org) to adopt it, given S4TF's demise, the likelihood that the original authors will abandon it, and the need for good benchmarking utilities that work across platforms.

**Comment by natecook1000** [comment]
Agreed on this — let's discuss here before going down an implementation path.

**Comment by stevapple** [comment]
Is the newly released [swift-collections-benchmark](https://github.com/apple/swift-collections-benchmark) suitable for such case? I thought that’s exactly the one we need.
